storehaus-postgresql/src/main/scala/com/twitter/storehaus/postgres/package.scala

```diff
@@ -0,0 +1,79 @@
+package com.twitter.storehaus
+
+import com.twitter.bijection.Injection
+import com.twitter.finagle.postgres.Client
+import com.twitter.util.Future
+
+
+import scala.util.Try
+
+/**
+  * @author Alexey Ponkin
+  */
+package object postgres {
+
+  type PostgresValueConverter[A] = Injection[A, PostgresValue]
+  type DBTxRequest[A] = Client => Future[A] // request in transaction
+
+  trait OneWayInjection[T] extends Injection[T, PostgresValue] {
+    override def apply(t: T): PostgresValue =
+      throw IllegalConversionException(
+        s"Can`t convert ${t.getClass.getCanonicalName} to PostgresValue")
+  }
+
+  implicit val boolInjection= new OneWayInjection[Boolean] {
+    override def invert(b: PostgresValue): Try[Boolean] = b match {
+      case Column(element) => Try(element.value.asInstanceOf[Boolean])
+    }
+  }
+
+  implicit val doubleInjection = new OneWayInjection[Double] {
+    override def invert(b: PostgresValue): Try[Double] = b match {
+      case Column(element) => Try(element.value.asInstanceOf[Double])
+    }
+  }
+
+  implicit val floatInjection = new OneWayInjection[Float] {
+    override def invert(b: PostgresValue): Try[Float] = b match {
+      case Column(element) => Try(element.value.asInstanceOf[Float])
+    }
+  }
+
+  implicit val shortInjection = new OneWayInjection[Short] {
+    override def invert(b: PostgresValue): Try[Short] = b match {
+      case Column(element) => Try(element.value.asInstanceOf[Short])
+    }
+  }
+
+  implicit val intInjection = new OneWayInjection[Int] {
+    override def invert(b: PostgresValue): Try[Int] = b match {
+      case Column(element) => Try(element.value.asInstanceOf[Int])
+    }
+  }
+
+  implicit val longInjection = new OneWayInjection[Long] {
+    override def invert(b: PostgresValue): Try[Long] = b match {
+      case Column(element) => Try(element.value.asInstanceOf[Long])
+    }
+  }
+
+  implicit val stringInjection = new OneWayInjection[String] {
+    override def invert(b: PostgresValue): Try[String] = b match {
+      case Column(element) => Try(element.value.asInstanceOf[String])
+    }
+  }
+
+  implicit val byteaInjection = new OneWayInjection[Array[Byte]] {
+
+    override def invert(b: PostgresValue): Try[Array[Byte]] = b match {
+      case Column(element) => Try(element.value.asInstanceOf[Array[Byte]])
+    }
+  }
+
+  def toEscString(a: Any): String = a match {
```

Comment:
I don't like this approach since we have no guarantee that `toString` is going to work. Why can't use a full injection to `PostgresValue` then use some safe mechanism to go from `Value[Any]` to `String` that can be interpolated in a SQL statement. I assume the `Value` wrapper has some support for that, no?